apache-maven/src/assembly/shared/validate.cmd

```diff
@@ -24,6 +24,7 @@
 @REM   MAVEN_BATCH_ECHO  (Optional) Set to 'on' to enable the echoing of the batch commands.
 @REM   MAVEN_BATCH_PAUSE (Optional) set to 'on' to wait for a key stroke before ending.
 @REM   MAVEN_OPTS        (Optional) Java runtime options used when Maven is executed.
+@REM   MAVEN_ARGS        (Optional) Arguments passed to Maven before CLI arguments.
```

Comment:
Assuming these are in alphabetic order, this is not the right location

Reply:
While you may be right I think they sorted logically by usage. I make it alphabetically, no issue.